data/scopes/fanvil-X210.ini

```diff
@@ -1,15 +1,16 @@
 [metadata]
 scopeType = "model"
 displayName = "Fanvil X210"
-version = 5
+version = 9
 
 [data]
 cap_ldap_tls_blacklist =
 provisioning_url_filename = "$mac.cfg"
 tmpl_phone = "fanvil-X5.tmpl"
-fanvil_sidekey_count = 10
+cap_linekey_count = 96
 fanvil_lkpages_count = 3
-cap_linekey_count = 106
```

Comment:
Changing this parameter is not possible as it breaks the NethVoice UI contract. For Fanvil U series phones `cap_linekey_count` represents the sum of side keys and line keys: they are represented by a linear array of Line Keys in the UI.

https://nethesis.github.io/tancredi/variables/#cap_linekey_count